src/node/rpc/node_frontend.h

```diff
@@ -771,7 +787,8 @@ namespace ccf
         auto nodes = args.tx.ro(this->network.nodes);
         nodes->foreach([this, host, port, status, &out](
                          const NodeId& nid, const NodeInfo& ni) {
-          const auto& primary_interface = ni.rpc_interfaces[0];
+          // TODO: Only match on the first interface?
```

Comment:
Not sure about this one: we used to return the node(s) that matched a given host/port/status/filter. I'd lean towards adding a new RPC interface name filter (optional) here and try to match it too?

Reply:
I will add the new interface filter in a follow-up PR.